### DISCUSSION TITLE
Why pnpm-workspaces.yaml?

### DISCUSSION THREAD

**Discussion by zhendalf** [pnpm/pnpm#3421] [Q&A] [answered]
Why pnpm workspaces are being defined in their own file, instead of "workspaces" field in package.json? 

It would be useful to have the reason it spelled out in the documentation. 

If this wouldn't be a breaking change, I'd prefer to have my workspaces defined in root package.json. Is this an option?

**Comment by zkochan** [comment, marked as answer]
There is a discussion about it already
https://github.com/pnpm/pnpm/discussions/3205
